### PR TITLE
feat(emul-12): Callout

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
## [EMUL-12: Callout](https://fourkitchens.atlassian.net/browse/EMUL-12)

**This PR does the following:**

- Creates the generic "Band" wrapper component
- Creates the initial Callout component
- Implements both in Storybook
- Updates the Gatsby homepage content to use the new Callouts

### Functional Testing:

- [ ] Run `yarn storybook` and checkout:
  - [ ] The [Band component](http://localhost:6006/?path=/story/molecules-band--band)
  - [ ] The [Callout component](http://localhost:6006/?path=/story/molecules-callout--callout)
  - [ ] The [Homepage example](http://localhost:6006/?path=/story/pages-homepage--homepage)
- [ ] Run `yarn develop` and verify everything above on [the Gatsby site](http://localhost:8000/).

**Notes:**
- The footer is shown, but that's coming form the `Layout` component. I'll be updating those styles etc. next.

### CODE REVIEW

**Comment by ryanhagerty** [comment]
@ModulesUnraveled I get the same errors as the previous PR - not sure if that's expected or to be addressed.

File: src/pages/index.tsx [hunk 1]
```diff
@@ -6,16 +6,18 @@ import { graphql } from 'gatsby'
...
-import SEO from '../components/base/seo/seo'
+// import SEO from '../components/base/seo/seo'
 
-import Card from '../components/molecules/card/card'
-import Features from '../components/organisms/features/features'
-import Signup from '../components/organisms/signup/signup'
-import { CtaGrid } from '../components/organisms/ctaGrid/CtaGrid'
+// import Card from '../components/molecules/card/card'
+// import Features from '../components/organisms/features/features'
+// import Signup from '../components/organisms/signup/signup'
```
Comment (ryanhagerty) [inline src/pages/index.tsx:13]: Are these TODOs or should they be removed?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -38,8 +40,23 @@ class RootIndex extends React.Component {
...
+          <Callout
+            layout="media-end"
+            heading="Give your developer and designers powerful tools all in one place."
+            text="Emulsify doesn’t just conveniently join your component library and style guide, but includes all of the workflow tools necessary for prototyping, testing, checking accessibility, and documenting."
+            image={
+              <Img fluid={this.props.data.devTools.childImageSharp.fluid} />
+            }
+          />
+        </Band>
+        {/* <SEO title="Home" url={this.props.location.href} /> */}
```
Comment (ryanhagerty) [inline src/pages/index.tsx:58]: Same question here

**Review by ryanhagerty — COMMENTED** [review]
This looks good to me. I'm getting some console errors/warnings related to Gatsby Link in the demo site. I'm cool with ignoring those for now if you just want to get this stuff in, but here they are:

`Failed prop type: The prop `to` is marked as required in `GatsbyLink`, but its value is `undefined`.`

And a couple warnings related to using external links with Gatsby Link, which should only be used for internal links.

File: src/pages/index.tsx (hunk 1 above)
Comment (ModulesUnraveled) [reply src/pages/index.tsx:13]: Yes, I'm leaving the old content for now. I'll remove all commented code once the homepage is completed with the new content/designs

**Comment by ModulesUnraveled** [comment]
@ryanhagerty I think I fixed all of the Link issues, as well as one about a missing key for the Ctas. Thanks for the review!

**Pull request merged**